Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md

```diff
@@ -0,0 +1,15 @@
+# Organizations 
+On Umbraco Cloud it is possible to be part of an organization. And organization is a way for you to associate users to one organization.
+The benefits are currently limited, but we are working on improving the features available for organizations.
+Currently organizations are created by Umbraco HQ - so **reach out to support** to hear more about the feature, and weather they apply for your scenario.
+
+## Invite organization to collaborate
+Once you have an organization you will be able to invite other organizations to collaborate for you.
+By inviting an organization the following is will happen
+* You allow a superadmin in that organization to create projects on behalf of your organization
+* You will be able to choose from super admins in the invited organization, when adding members to a project.
+  * This effectly enalbes the super admin in that organization access to your project, but also allows her to invite other member from that organization.
```

Comment:
Typos here:
effectly --> effectively
enalbes --> enables
access to --> to access
member --> members

Fixed line:
 This effectively enables the super admin in that organization to access your project, but also allows her to invite other members from that organization.